pydm/widgets/timeplot.py

```diff
@@ -1,36 +1,53 @@
-from pyqtgraph import CurvePoint
 import time
 import json
 from collections import OrderedDict
 from pyqtgraph import ViewBox, AxisItem
 import numpy as np
 from qtpy.QtGui import QColor
 from qtpy.QtCore import Qt, Slot, Property, QTimer
+from qtpy.QtWidgets import QAction
 from .baseplot import BasePlot, BasePlotCurveItem
 from .channel import PyDMChannel
 from .. utilities import remove_protocol
 
+import logging
+logger = logging.getLogger(__name__)
+
 MINIMUM_BUFFER_SIZE = 10
 DEFAULT_X_MIN = -30
```

Comment:
Which unit is the `-30` and how does it affect the overall plot?

Reply:
The default view (initially, and whenever the Reset zoom button is pressed) is to show the chart with the current time as 0 at the right most position, and the most recent 30 seconds past. That's why I set the DEFAULT_X_MIN value to -30.